src/modules/app/SentryIntegration.js

```diff
@@ -24,7 +24,7 @@ export default class SentryIntegration {
       return
     }
 
-    this.sentry.install()
+    await this.sentry.install()
```

Comment:
Not sure whether we need/want to wait on this if it is not needed. The sentry API behaves a little bit unexpected.

Official docs do not await it: https://docs.sentry.io/clients/react-native/#configure

Reply:
Well otherwise it's not possible catch any errors that happen when installing it (see https://github.com/Integreat/integreat-react-native-app/blob/4ded4229bcd35aaa2949e02bed290f82f03b2e26/src/modules/app/components/LaunchInquiry.js#L84). Also the example you refer to is very minimalistic I guess...